docroot/profiles/humsci/su_humsci_profile/su_humsci_profile.post_update.php

```diff
@@ -39,3 +39,47 @@ function su_humsci_profile_post_update_8_0_4() {
     $block->delete();
   }
 }
+
+/**
+ * Fix layout builder block display.
```

Comment:
What are you fixing? Please leave a more detailed description.

Reply:
what you cant tell from the short ambigous comment?